deployment/docker/traffic-replayer/run_app.sh

```diff
@@ -10,22 +10,22 @@ do
   # Verify connection with TC can be made before disconnecting via 1 second timeout and moving forward
   while ! nc -v -w 1 localhost 9220
   do
-    echo 'Attempting to connect to Traffic Comparator...'
+    >&2 echo 'Attempting to connect to Traffic Comparator...'
     sleep 2
   done
 
   # Allow time for TC to begin listening after terminated connection
-  sleep 10
+  sleep 7
 
   touch triples.log
 
   # These commands look a bit backwards but ultimately will receive input from CW-Puller via netcat and then
   # process with the Replayer jar before ultimately printing the produced triples to stdout via tee and sending to TC via netcat
   # Note: Unbuffer has not been verified to actually be effective here. stdbuf was ruled out for its caveat with tee (https://linux.die.net/man/1/stdbuf)
   unbuffer tail -f triples.log | tee /dev/stderr | nc -v localhost 9220 &
```

Comment:
Why are you using a file at all here?  Why not pipe directly?  Does unbuffer matter for the case where lots of messages are going through?

Reply:
I believe at the time we were running into issue with having netcat receiving and sending in the same line. That may not be present now, but ideally we move away from netcat anyway